.github/workflows/test-pr-pipeline.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Install dependencies, Lint with black, & run tests with Pytest
         working-directory: "06-best-practices/code"
         run: |
-          pip install pipenv && pipenv install --dev && pip install -r requirements.txt
+          pip install pipenv && pipenv install --dev && pipenv run pylint --recursive=y .
           make quality_checks
```

Comment:
I'd do this as separate steps. First install dependencies, then run pytest and then pylint (or first pylint then tests, doesn't matter). All 3 separate steps